docs/subset14.md

```diff
@@ -234,13 +234,13 @@ simple-const-expr =
    | "-" floating-point-literal
    | const-reference-expr
 
-expression = inner-expr | list-constructor-expr | mapping-constructor-expr
+expression = logical-or-expr
 
-const-expr = inner-expr # must also satisfy restrictions of const-expr as in Ballerina language spec
+const-expr = logical-or-expr # must also satisfy restrictions of const-expr as in Ballerina language spec and should be neither list-constructor-expr nor mapping-constructor-expr
```

Comment:
Don't need this, given that it is mentioned below.

Reply:
fixed